test/distributed/org/apache/cassandra/distributed/test/RepairRequestTimeoutTest.java

```diff
@@ -45,7 +45,9 @@ public class RepairRequestTimeoutTest extends TestBaseImpl
     public static void setup() throws IOException
     {
         CLUSTER = init(Cluster.build(3)
-                              .withConfig(config -> config.with(GOSSIP, NETWORK).set("repair_request_timeout_in_ms", timeoutMillis))
+                              .withConfig(config -> config.with(GOSSIP, NETWORK)
+                                                          .set("repair_request_timeout_in_ms", timeoutMillis)
+                                                          .set("repair.retries.max_attempts", 1))
```

Comment:
Aiming to make this test less flaky?

Reply:
yeah, it was flakey... but can revert, it was mostly to get things stable while I was refactoring... for now ill delete...

if my patch makes this test flakey... thats a bug